public/js/components/App.js

```diff
@@ -1,47 +1,47 @@
 class App extends React.Component{
    state={
-       shoppingCart:[]
+       shoppingCart:[],
+       view: "products"
    }
    addItemToCart = (product)=> {
     this.setState(()=>{
         this.state.shoppingCart.push(product);
         return {shoppingCart:this.state.shoppingCart}
     })
    }
+   changeView = (page)=> {
+       return page === "products" ? 
+       this.setState({
```

Comment:
double check this syntax for setState. I think you are mixing up the 2 different ways to do it